registry/data/third-party.json

```diff
@@ -162,6 +149,19 @@
         "cosign"
       ]
     },
+    {
+      "id": "cue",
+      "locator": "https://raw.githubusercontent.com/appthrust/proto-toml-plugins/main/cue/plugin.toml",
+      "format": "toml",
+      "name": "CUE",
+      "description": "CUE makes it easy to validate data, write schemas, and ensure configurations align with policies.",
+      "author": "appthrust",
+      "homepageUrl": "https://github.com/appthrust/proto-toml-plugins",
+      "repositoryUrl": "https://github.com/appthrust/proto-toml-plugins/tree/main/cue",
+      "bins": [
+        "cue"
+      ]
+    },
```

Comment:
**Note:** This change is not related to the main purpose of this PR. It was automatically fixed by `just gen`.